source/common/grpc/buffered_message_ttl_manager.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <queue>
+
+#include "envoy/event/dispatcher.h"
+
+#include "source/common/grpc/buffered_async_client.h"
+
+namespace Envoy {
+namespace Grpc {
+
+class BufferedMessageTtlManager {
```

Comment:
I think this class could benefit from some comments, it's not immediately clear how this works or what it tries to solve

Reply:
Added